src/main/java/com/yetanalytics/xapi/model/XapiDuration.java

```diff
@@ -7,9 +7,19 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
+/**
+* Class representation of ISO 8601 Duration allowing for retrieval of the 
+* original String.
+*/
```

Comment:
You can add a link to the duration definition in the xAPI spec: https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#46-iso-8601-durations

Reply:
88e7b0911a596e7759917cc2c895b85de9a2eb82